docs/reference/authenticated_authors.rst

```diff
@@ -47,30 +42,20 @@ Open entity ``Application\Sonata\CommentBundle\Entity\Comment`` and implement th
             return $this->id;
         }
 
-        /**
-         * {@inheritdoc}
-         */
         public function setAuthor(UserInterface $author)
```

Comment:
Do we need these methods? Don’t they exist in the BaseComment class?

Reply:
Not sure but this is more a cleanup. We can fix this in an extra PR